src/main/java/com/spotify/docker/client/messages/Ipam.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2014 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.docker.client.messages;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+
+@JsonAutoDetect(fieldVisibility = ANY, getterVisibility = NONE, setterVisibility = NONE)
+public class Ipam {
+
+  public Ipam() {
```

Comment:
removed

2015-12-28 18:20 GMT+01:00 David Xia <notifications@github.com>:

> In src/main/java/com/spotify/docker/client/messages/Ipam.java
> <https://github.com/spotify/docker-client/pull/307#discussion_r48492539>:
>
> > +
> > +import com.fasterxml.jackson.annotation.JsonAutoDetect;
> > +import com.fasterxml.jackson.annotation.JsonProperty;
> > +import com.google.common.base.MoreObjects;
> > +import com.google.common.base.Objects;
> > +
> > +import java.util.ArrayList;
> > +import java.util.List;
> > +
> > +import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
> > +import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
> > +
> > +@JsonAutoDetect(fieldVisibility = ANY, getterVisibility = NONE, setterVisibility = NONE)
> > +public class Ipam {
> > +
> > +  public Ipam() {
>
> Looks like we don't need this constructor.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/docker-client/pull/307/files#r48492539>.
>



-- 
Damiano Pezzotti
...................................... . .
Software Architect | Alambitco
mobile +39 333 2116258
fax +39 02 700432209
email damy@alambitco.com
web www.alambitco.com
flickr www.flickr.com/photos/damianopezzotti
linkedin www.linkedin.com/in/damianopezzotti
pgp id 0x12DFD989